DirectProgramming/DPC++FPGA/include/unrolled_loop.hpp

```diff
@@ -87,7 +87,7 @@ constexpr void UnrolledLoop(F&& f) {
 }
 
 //
-// Convenience implementation (C)
+// Convenience implementation (D)
```

Comment:
What this intended?

Reply:
Yes it was.  It's not really related to this change, just something I noticed while working on this change, a simple error in the comments of this file.  I could put it into a separate PR if you want, but it's small enough I was hoping to just sneak it through with this one . . .

Reply:
I'm fine with the sneaking :)